Doc/library/urllib.request.rst

```diff
@@ -608,21 +608,23 @@ OpenerDirector Objects
 
    *handler* should be an instance of :class:`BaseHandler`.  The following methods
    are searched, and added to the possible chains (note that HTTP errors are a
-   special case).
+   special case). Note that, in the following, *protocol* should be replaced
+   with the actual protocol to handle, for example :meth:`http_response` would
+   be the HTTP protocol response handler.
 
-   * :meth:`protocol_open` --- signal that the handler knows how to open *protocol*
+   * :meth:`<protocol>_open` --- signal that the handler knows how to open *protocol*
      URLs.
 
    * :meth:`http_error_type` --- signal that the handler knows how to handle HTTP
      errors with HTTP error code *type*.
```

Comment:
The contrast with _http_error_type_ now makes it less obvious that _type_ is also a placeholder. I suggest to keep the notation uniform.